tests/saw/spec/handshake/s2n_handshake_io.cry

```diff
@@ -141,7 +148,7 @@ IS_TLS13_HANDSHAKE conn = conn.actual_protocol_version == S2N_TLS13
 ACTIVE_STATE_MACHINE : connection -> [19]handshake_action
 ACTIVE_STATE_MACHINE conn = if IS_TLS13_HANDSHAKE conn then tls13_state_machine else state_machine # zero
 
-ACTIVE_HANDSHAKES : connection -> [128][32][32]
+ACTIVE_HANDSHAKES : connection -> [256][32][32]
```

Comment:
We should probably define the number of handshakes as a standalone type, since it's used so many places.